apps/src/utils.js

```diff
@@ -559,14 +568,45 @@ export function normalize(vector) {
   };
 }
 
+type Position =
+    1
+  | 2
+  | 3
+  | 4
+  | 5
+  | 6
+  | 7
+  | 8
+  | 9
+  | 10
+  | 11
+  | 12
+  | 13
+  | 14
+  | 15
+  | 16
+  | 17
+  | 18
+  | 19
+  | 20
+  | 21
+  | 22
+  | 23
+  | 24
+  | 25;
```

Comment:
Yeah, you can use `$Enum`, though `$Values` is what I'd really want, as proposed here: https://github.com/facebook/flow/issues/961. I hear it's coming.